configuration.k

```diff
@@ -0,0 +1,220 @@
+            // so that the deployed contract have access to them and can create and deploy them if needed. 
+            // (see the <account> cell below for more notes about this)
+
+            <contracts>
+                <Contract multiplicity = "*">
+                    <cname> .K </cname>
+                    
+                    <functions>
+                        <Function multiplicity="*">
+                            <fname>         .K         </fname>
+                            <inputParams>  .K         </inputParams> 
+                            <Returns>      .K         </Returns>
+                            <quantifiers>  .K         </quantifiers>
+                            <visibility>   .K         </visibility> // Visibility ::= public | privat
+                            <body> .K </body>
+                        </Function>
+                    </functions>
+		            
+                    <stateVar> .List </stateVar>
+                    <Constructor> false </Constructor>
+                    // ... more to be added later ...
+                
+                </Contract>
+            </contracts>
+
+        </solidity>
+
+        // Ethereum Network
+        // ==========================
+
+        <network>
+
+            // Accounts Record
+            // ---------------
+
+            <activeAccounts> .Set </activeAccounts>
+	        <cntAccount> 0:Int </cntAccount>
+            <accounts>
+                <account multiplicity="*">
+                    
+                    // Same as in KEVM / IELE
+                    
+                    <acctID>  .K                      </acctID>
+                    <balance> 0                      </balance>
+		            <context> .Map		     </context>
+                    
+                    // When a contract is deployed, its <contract> data structure will be copied here.
+                    // Because contracts can create other contracts (via "new") we need to also keep the 
+                    // set of contract that this contract can create (i.e. the set of contracts that was
+                    // available at creation time, like in IELE)
+
+                    <code> 
+                        <associatedContract> .K </associatedContract>    // contract deployed on this account
+                        <availableContracts> .K </availableContracts>    // contracts that can be created by this account's contract
+                    </code>
+
+                    // [Jerry's notes about storage]:
+                    // this is exactly same as evm
+                    // Solidity has rules to allocate indices/offset(packing) to each storage
+                    // variable and I think these rules should be the part of Solidity semantics,
+                    // so that the storage usage of solidity program and evm bytecode have
+                    // the same effect on the storage.
+                    
+                    <Storage> .Map  </Storage>
```

Comment:
why is this capitalized?

Reply:
Initially, we use `<storage>`, but end up with a compilation error (kompile) saying that `<storage>` conflicts with the Solidity keyword `storage`. Could you advise how to solve this issue?

Reply:
@shangweilin you won't be able to use `storage` then, I would follow the same convention and do `acctStorage`.

Reply:
Fixed.